packages/sst/build.mjs

```diff
@@ -1,7 +1,14 @@
 import esbuild from "esbuild";
 import fs from "fs/promises";
 
-const pkg = await fs.readFile("package.json").then(JSON.parse);
+// Copy package.json to dist, excluding publishConfig
```

Comment:
I'm not really sure why dist/package.json didn't have a `publishConfig` section. I assumed there was a reason, so it's being excluded when copying package.json to dist/package.json.